packages/components/src/toggle-group-control/toggle-group-control/as-radio-group.tsx

```diff
@@ -73,15 +74,24 @@ function UnforwardedToggleGroupControlAsRadioGroup(
 	const setValue = radio.setValue;
 
 	const groupContextValue = useMemo(
-		() =>
-			( {
-				baseId,
-				isBlock: ! isAdaptiveWidth,
-				size,
-				value: selectedValue,
-				setValue,
-			} ) as ToggleGroupControlContextProps,
-		[ baseId, isAdaptiveWidth, size, selectedValue, setValue ]
+		(): ToggleGroupControlContextProps => ( {
+			baseId,
+			isBlock: ! isAdaptiveWidth,
+			size,
+			// @ts-expect-error - This is wrong and we should fix it.
```

Comment:
This mismatch between `null` and `undefined` as possible `value` types was already there prior to this PR, and it sounds like something we should solve — cc @mirka , could this be part of the work you're intending to do about fixing [ToggleGroupControl's focus bug](https://github.com/WordPress/gutenberg/issues/62981)?